ipaserver/plugins/internal.py

```diff
@@ -250,6 +250,10 @@ class i18n_messages(Command):
             "update": _("Update"),
             "view": _("View"),
         },
+        "customization": {
+            "customization": _("Customization"),
+            "table_pagination": _("Table Pagination Size"),
```

Comment:
What about using only "Pagination size". "Table" won't tell much to users. I.e. "what table?"